src/shady/passes/spirv_map_entrypoint_args.c

```diff
@@ -0,0 +1,84 @@
+#include "passes.h"
+
+#include "portability.h"
+#include "log.h"
+
+#include "../rewrite.h"
+#include "../type.h"
+#include "../transform/ir_gen_helpers.h"
+#include "../transform/memory_layout.h"
+
+typedef struct {
+    Rewriter rewriter;
+    CompilerConfig* config;
+} Context;
+
+static Nodes append_annotations(Rewriter* rewriter, Nodes orig_annotations) {
```

Comment:
use `append_nodes` from `ir.h`

Reply:
seems like this fn does nothing rn

Reply:
I'll just remove this since it's unused now anyways.